lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1325,6 +1326,13 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => true,
           :description => 'If `true`, the agent obfuscates Elasticsearch queries in transaction traces.'
         },
+        :'elasticsearch.perform_health_check' => {
+          :default => true,
+          :public => true,
+          :type => Boolean,
+          :allowed_from_server => true,
```

Comment:
There is nothing on the server side that would update this config, so this should probably be false.